src/btf.cpp

```diff
@@ -152,27 +153,26 @@ void BTF::load_module_btfs(const std::set<std::string> &modules)
     }
 
     // Get BTF object info - needed to determine if this is a kernel module BTF
-    char name[64];
+    char name[64] = { 0 };
```

Comment:
Nit: `={ 0}` technically initializes the first element to 0 and the rest, b/c they're unnamed, to 0 as well. Dropping the `0` is more accurate.
```suggestion
    char name[64] = {};
```